commit f10131de05ae979cdc7333f5ccfecbeb27265785
Author: Yichao Yu <yyc1992@gmail.com>
Date:   Fri Nov 29 19:03:50 2013 -0500

    Do not encode QString to QByteArray and cast it back to QString
    
    This causes problem when there are Unicode characters in ${HOME}
    
    REVIEW: 114219
    BUG: 327919
    FIXED-IN: 4.11.5

diff --git a/kcontrol/krdb/krdb.cpp b/kcontrol/krdb/krdb.cpp
index 92d84e9..31e8820 100644
--- a/kcontrol/krdb/krdb.cpp
+++ b/kcontrol/krdb/krdb.cpp
@@ -99,7 +99,7 @@ static void applyGtkStyles(bool active, int version)
 
    // Pass env. var to kdeinit.
    QString name = gtkEnvVar(version);
-   QString value = QFile::encodeName(list.join(":"));
+   QString value = list.join(":");
    KToolInvocation::klauncher()->setLaunchEnv(name, value);
 }
 
